README.md

```diff
@@ -63,13 +65,17 @@ In the Arguments section, add five (or more) Arguments Passed On Launch to the b
   ```
   --base-path="/Users/YourName/Desktop/YourProjectFolder"
   ```
-- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs:  
+- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs):  
   ```
   --asset-catalog="Resources/Images.xcassets"
   ```  
   ```
   --asset-catalog="Resources/Media.xcassets"
   ```
+- The prefix for the category method names. If not specified, the default is `ac`. Must be at least 2 characters, and conform to valid method naming rules:
```

Comment:
I'm wondering if maybe we're making this too complicated—maybe `Cat2Cat` should get invoked separately for each prefix?  That is, if you've got some stuff with prefix `abc` and some with prefix `xyz`, you should run `Cat2Cat` 2 separate times?  That'd avoid all the issues about aligning different asset catalogs with different prefixes.